### PR TITLE
Add applicant thumbnail component

### LINKED ISSUE #18: Applicant Component
**Describe the feature to implement.**

Implement the applicant component. The component should have an accept and deny button and have some control to allow it to change to accepted or denied.

**Figma Screenshot if applicable**

image

**Additional context if applicable**

### PR DESCRIPTION
**Pull request by AmanKAggarwal**
### Adminstrative Info
[closes #18]

### Changes
What changes did you make?
- Added applicant thumbnail component
### Testing
How did you confirm your changes worked?
- Manually tested it
### Confirmation of Change
screenshot

### CODE REVIEW

File: LAKMobile/src/components/ApplicantThumbnail.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+export function ApplicantThumbnail({applicantData, status, style}: ApplicantThumbnailProps) {
+    const flexJustifyContent = 'space-between'
+    return (
+        <View style={[ThumbailStyles.container, style]}>
+            <AppText style={[ThumbailStyles.nameFont, ThumbailStyles.containerItem]}>{applicantData.firstName} {applicantData.lastName}</AppText>
+            <AppText style={[ThumbailStyles.phoneFont, ThumbailStyles.containerItem]}>{applicantData.phone}</AppText>
+            <AppText style={ThumbailStyles.containerItem}>{applicantData.vehicleInformation}</AppText>
+
+            <View style={[ThumbailStyles.flex, status == 'Accepted' ? {justifyContent: 'space-between'} : null]}>
+                {status == 'Unassigned' && <AppButton type="tertiary" size="small" title="Accept" style={ThumbailStyles.flexItem} textStyle={{color: COLORS.turquoise}}/>}
```
Comment (NirmalAgnihotri) [inline LAKMobile/src/components/ApplicantThumbnail.tsx:25]: Good formatting on the different conditions

**Review by NirmalAgnihotri — APPROVED** [review]
Looks great and code is well formatted

**Comment by AmanKAggarwal** [comment]
`Reject` -> `Denied`

**Comment by AmanKAggarwal** [comment]
image

**Pull request merged**